README

```diff
@@ -296,6 +296,13 @@ Description:	If enabled, the response from the CAS Server will be parsed for SAM
 		CASValidateURL appropriately; typical URLs are of the form
 		https://login.example.org/cas/samlValidate.
 
+Directive:	CASAllowSubAuth
+Default:	Off
+Description:	This directive determines whether subsequent CAS authentication
+		responses are allowed to pass through, leaving the ticket parameter intact.
```

Comment:
I don't like the phrase "CAS authentication responses". 

Is something like the following more accurate:

```This directive leaves CAS ticket parameters intact when a valid session cookie exists. This helps prevent infinite redirect loops when CAS protection is being used at multiple levels.```